app/models/concerns/profile_searching.rb

```diff
@@ -4,11 +4,17 @@
 module ProfileSearching
   extend ActiveSupport::Concern
 
+  OPERATORS = { '=' => :eq, '<>' => :ne, 'LIKE' => :like, 'ILIKE' => :like,
+                'NOT LIKE' => :unlike, 'NOT ILIKE' => :unlike,
+                '>' => :gt, '<' => :lt, '<=' => :lte, '>=' => :gte,
+                'IN' => :in, 'NOT IN' => :notin }.freeze
```

Comment:
Aren't these defined somewhere in scoped search? Then you don't need to define them twice, just refer the constant below from there and drop this.

Reply:
The constant in scoped search is the same as this but the keys/values are opposite. So it's `:eq => '='` instead of `'=' => :eq`. And that constant doesn't include `ILIKE` or `NOT ILIKE` :confused:

Reply:
Let's extract these into the `ScopedSearch::QueryBuilder` monkeypatch we have in the initializers.

Reply:
@marleystipich2 please don't forget about this one